docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+services:
+  app:
+    build:
+      context: .
+      dockerfile: src/main/docker/Dockerfile
+    env_file:
+      - .env
+    depends_on:
+      db:
+        condition: service_healthy
+
+  db:
+    image: postgres:14.1-alpine
+    restart: always
+    env_file:
+      - .env
+    healthcheck:
+      test: ["CMD", "pg_isready", "-h", "${POSTGRES_HOST}", "-p", "5432", "-U", "${POSTGRES_USER}", "-d", "${POSTGRES_DB}"]
```

Comment:
We were seeing the error "service unhealthy", while docker-compose was setting up the database because I mistakenly wrote "-u" instead of "-U". This caused healthcheck to fail every time with an invisible "unknown flag" command I think.

Reply:
well done!